src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -100,6 +108,9 @@
     private OCFile mFile = null;
     private FileListListAdapter mAdapter;
 
+    private LinearLayout filesLinearLayout;
```

Comment:
I would rename this layout as uploadFilesLinearLayout so that is not misunderstood as the list of files.